cognite/neat/rules/models/entities/_multi_value.py

```diff
@@ -0,0 +1,67 @@
+from typing import Any, ClassVar
```

Comment:
I guess this is to make some distinction between previous _base, which is turned into _signel_value.py